### ISSUE TITLE
Doesnt show the translation / exaile in english

### ISSUE THREAD

**Issue by psyca** [exaile/exaile#54]
Somehow exaile doest take the translated .po files.
I tried the Windows installation (german) and the text in exaile is still in english.
If i look in the po folder, the german translation looks ok.
Same i found on linux.

**Comment by virtuald** [comment]
The option for the installer's language is different than what Exaile actually uses. Exaile will use the detected locale -- so if your OS is in English locale, it will use that.

When exaile 3.4.x first starts from the command line, it says something like "Using XXX locale". What does your installation of Exaile say?

**Comment by psyca** [comment]
INFO    : Loading Exaile 3.4.2 on Python 2.7.9...
INFO    : Loading settings...
INFO    : Using de_DE UTF-8 locale
** Message: pygobject_register_sinkfunc is deprecated (GstObject)
INFO    : Loading plugins...
INFO    : Loading collection...
WARNING : Exception occurred while loading /home/jurgen/.local/share/exaile/music.db
WARNING : Exception occurred while loading /home/jurgen/.local/share/exaile/music.db
WARNING : Exception occurred while loading /home/jurgen/.local/share/exaile/music.db
INFO    : Loading devices...
INFO    : Connected to udisks2
INFO    : Loading interface...
INFO    : Using GTK+ 2.24.26
INFO    : Loading main window...
/usr/share/exaile/xlgui/guiutil.py:123: Warning: g_object_set_qdata: assertion 'G_IS_OBJECT (object)' failed
  parent.add(replacement)
INFO    : Connecting main window events...
INFO    : Loading panels...
INFO    : Connecting panel events...
INFO    : Done loading main window...
INFO    : WARNING: could not open podcast file
INFO    : Exaile is shutting down...
INFO    : Disabling plugins...
INFO    : Saving state...
INFO    : Bye!
Exception TypeError: "'NoneType' object is not callable" in  ignored
Exception TypeError: "'NoneType' object is not callable" in  ignored

----

This problem was also discussed in the german Mageia forum:
https://forums.mageia.org/de/viewtopic.php?f=18&t=2411

**Comment by virtuald** [comment]
Ah, the information on the forum is useful, I see the problem. I'll look into fixing it tonight.

**Comment by virtuald** [comment]
There's definitely a problem with the way that Exaile installs it's locale files from the Makefile. It installs them by default to /usr/local/share/locale/po, but gettext is looking for them (seen via strace) in /usr/share/locale/po.

As for the problem reported on mageia, it seems like it installs exaile a bit differently than I would expect. I can't seem to find their RPM repository, can you post the output of `rpm -ql exaile` to a gist/pastebin? Thanks!

**Comment by psyca** [comment]
http://www.mageia.psyca.de/exaile.txt

**Comment by virtuald** [comment]
Cool, just as I suspected. Looks like Mageia packaging exaile in `/usr/share/exaile` -- with the exception of the po files, which it places in a standard location. The code that tries to detect whether we're running from a local checkout looked for a 'data' directory -- which existed, but not the 'po' directory. I changed it to search for the 'po' directory, which should solve that particular problem for you.

However, does not fix some secondary problems yet.
